setup.py

```diff
@@ -34,6 +34,11 @@
         'License :: OSI Approved :: BSD License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.4',
```

Comment:
Why do you want to add support for 3.4 which is [EOLed](https://devguide.python.org/#status-of-python-branches) already?

Reply:
I wasn't aware it EOLed and  so I didn't want to remove support for it...
but since it is EOLed, it makes sense to remove it.
done.